contributing-guides/alias-pages.md

```diff
@@ -0,0 +1,170 @@
+# Alias pages
+
+In order to document a command which is an alias of another command, you can
+write an alias page. This file contains a list of all translations of the alias
+page template decided upon in
+[#5368](https://github.com/tldr-pages/tldr/pull/5368).
+The templates can be changed when necessary.
+
+[en](#en) •
+[bs](#bs) •
+[da](#da) •
+[de](#de) •
+[es](#es) •
+[fa](#fa) •
+[fr](#fr) •
+[hi](#hi) •
+[id](#id) •
+[it](#it) •
+[ja](#ja) •
+[ko](#ko) •
+[ml](#ml) •
+[nl](#nl) •
+[no](#no) •
+[pl](#pl) •
+[pt_BR](#pt_br) •
+[pt_PT](#pt_pt) •
+[ru](#ru) •
+[sh](#sh) •
+[sv](#sv) •
+[ta](#ta) •
+[th](#th) •
+[tr](#tr) •
+[zh](#zh) •
+[zh_TW](#zh_tw)
+
+### en
+
+```Markdown
+# example
+
+> This command is an alias of `example`.
+
+- View documentation for the original command:
+
+`tldr example`
+```
```

Comment:
Hi @2KAbhishek! Yes, can we have a freshly crafted Hindi translation for this snippet (between the \`\`\`)?

Reply:
I'm on it!

Reply:
Just send a message right here...no need to open a PR or anything like that ;)

Reply:
```
# उदाहरण

> यह आदेश `उदाहरण` का उपनाम है।

- मूल आदेश के लिए दस्तावेज़ देखें:

`tldr उदाहरण`
```

Reply:
here you go @bl-ue